js/script.js

```diff
@@ -0,0 +1,230 @@
+		var tdId = document.createElement("td");
+		tdId.innerHTML = data[i].id;
+		tdId.classList.add("cellWithoutSize");
+		var tdNome = document.createElement("td");
+		tdNome.innerText = data[i].name;
+		var tdEmail = document.createElement("td");
+		tdEmail.innerText = data[i].email;
+		tr.appendChild(tdId);
+		tr.appendChild(tdNome);
+		tr.appendChild(tdEmail);
+		tr.dataset.id = data[i].id;
+		tr.onclick = function(){
+			httpRequestInfo = new XMLHttpRequest();
+
+			httpRequestInfo.open('GET', ' https://jsonplaceholder.typicode.com/users/'+this.dataset.id);
+			httpRequestInfo.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+			httpRequestInfo.onreadystatechange = function(){
+				if(httpRequestInfo.readyState === 4){
+					var dataInfo = JSON.parse(httpRequestInfo.responseText);
+					while(info.lastChild){
+						info.removeChild(info.lastChild);
+					}
+					objectToHTML(dataInfo);
+
+				}
+			};
+
+			httpRequestInfo.send();
+
+		}
+		tbody.appendChild(tr);
+	}
+}
+
+function mountTablePublication(data){
+	var thead = document.getElementById("users-thead");
+
+	while(thead.lastChild){
+		thead.removeChild(thead.lastChild);
+	}
+	var tbody = document.getElementById("users");
+	while(tbody.lastChild){
+		tbody.removeChild(tbody.lastChild);
+	}
+	while(info.lastChild){
+		info.removeChild(info.lastChild);
+	}
+
+	if(!data.length){
+		var tr = document.createElement("tr");
+		var tdNothig = document.createElement("td");
+		tdNothig.innerHTML = "Nenhum resultado encontrado."
+		tr.appendChild(tdNothig);
+		tbody.appendChild(tr);
+	}
+
+	for(var i = 0; i<data.length; i++){
+		if(!i){
+			var trHead = document.createElement("tr");
+			var thId = document.createElement("th");
+			thId.innerHTML = "ID";
+			thId.classList.add("cellWithoutSize");
+			var thTitulo = document.createElement("th");
+			thTitulo.innerHTML = "Titulo";
+			trHead.appendChild(thId);
+			trHead.appendChild(thTitulo);
+			thead.appendChild(trHead);
+		}
+
+		var tr = document.createElement("tr");
+		var tdId = document.createElement("td");
+		tdId.innerHTML = data[i].id;
+		tdId.classList.add("cellWithoutSize");
+		var tdTitulo = document.createElement("td");
+		tdTitulo.innerText = data[i].title;
+
+		tr.appendChild(tdId);
+		tr.appendChild(tdTitulo);
+		tr.dataset.id = data[i].id;
+		tr.onclick = function(){
+			httpRequestInfo = new XMLHttpRequest();
+
+			httpRequestInfo.open('GET', 'https://jsonplaceholder.typicode.com/posts/'+this.dataset.id);
+			httpRequestInfo.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+			httpRequestInfo.onreadystatechange = function(){
+				if(httpRequestInfo.readyState === 4){
+					var dataInfo = JSON.parse(httpRequestInfo.responseText);
+					while(info.lastChild){
+						info.removeChild(info.lastChild);
+					}
+					objectToHTML(dataInfo);
+
+				}
+			};
+
+			httpRequestInfo.send();
+
+		}
+		tbody.appendChild(tr);
+	}
+}
+
+function objectToHTML(data, property){
+	for(a in data){
```

Comment:
Add the "let", "const" or "var" keyword to this declaration of "a" to make it explicit.